components/eam/src/physics/cam/hetfrz_classnuc.F90

```diff
@@ -278,7 +224,6 @@ subroutine hetfrz_classnuc_calc( &
    !*****************************************************************************
    !   solute effect
    aw(:) = 1._r8
```

Comment:
maybe we can remove the initiation of this variable as it is the output of the subroutine call next.